components/performance-chart/styles.css

```diff
@@ -1,5 +1,6 @@
 .container {
   position: relative;
+  max-height: 300px;
```

Comment:
Does not a single limit placement fix the issue?

Reply:
Unfortunately not :(